core/src/main/resources/i18n/displayStrings.properties

```diff
@@ -3215,8 +3215,8 @@ payment.f2f.info='Face to Face' trades have different rules and come with differ
   ● The trading peers need to bring their laptops and do the confirmation of 'payment sent' and 'payment received' at the meeting place.\n\
   ● If a maker has special 'terms and conditions' they must state those in the 'Additional information' text field in the account.\n\
   ● By taking an offer the taker agrees to the maker's stated 'terms and conditions'.\n\
-  ● In case of a dispute the mediator or arbitrator cannot help much as it is usually hard to get tamper proof evidence of what \
-  happened at the meeting. In such cases the BTC funds might get locked indefinitely or until the trading peers come to \
+  ● In case of a dispute the mediator or arbitrator cannot be of much assistance as it is usually difficult to get tamper proof evidence \
```

Comment:
```suggestion
  ● In case of a dispute the mediator or arbitrator cannot be of much assistance as it is usually difficult to get tamper-proof evidence \
```